web-app/components/pages/login/login.jsx

```diff
@@ -82,6 +83,32 @@ export const Login = () => {
         },
         onSubmit: (values) => {
```

Comment:
A better code is destructuring the value like
```suggestion
        onSubmit: ({email, password}) => {
```